lms/static/sass/discussion/lms-discussion-bootstrap.scss

```diff
@@ -1,10 +1,12 @@
 @import 'lms/static/sass/discussion/lms-discussion-main';
+@import 'lms/static/sass/base/extends';
+@import 'lms/static/sass/course/layout/courseware_header';
+@import '../partials/lms/theme/variables-v1';
 @import '../overrides';
 
 // The discussion page uses pattern library v2 while most other pages use v1.
 // These tweaks are needed to make the discussion page look more consistent
 // with the rest of the site.
-@import 'lms/static/sass/course/layout/courseware_header';
```

Comment:
Is the comment above now referring to everything below this?

Reply:
That's correct, I think.